tests/test_price.py

```diff
@@ -0,0 +1,21 @@
+from decimal import Decimal
+
+import pytest
+from lxml.html import fromstring
+from price_parser import Price
+
+from zyte_parsers.price import extract_price
+
+
+@pytest.mark.parametrize(
+    ["html", "currency_hint", "expected"],
+    [
+        ("<p></p>", None, Price(None, None, None)),
+        ("<p>23.5</p>", None, Price(Decimal(23.5), None, "23.5")),
+        ("<p>$23.5</p>", None, Price(Decimal(23.5), "$", "23.5")),
+        ("<p>23.5</p>", "USD", Price(Decimal(23.5), "USD", "23.5")),
```

Comment:
I think it'd be nice to add a test for the case currency_hint is different from the currency in the html. It's tested in price-parser, but other cases are tested as well.

Reply:
Done, should I add some more?

Reply:
In price-parser currency_hint is defined as 

> ``currency_hint`` is optional; you can pass value of some element which may contain currency, as a hint.

It seems it'd be good to add a test where the value passed to currency_hint is not exact. 

It also could make sense to support a Selector or Node for currency_hint, btw.

Should we also copy more of the docstring here? currency_hint behavior is non obvious.